GUACAMOLE-1054: Improve Russian translation

darksimpson opened the pull request.

necouchman commented on extensions/guacamole-auth-duo/src/main/resources/translations/ru.json:
```diff
@@ -0,0 +1,13 @@
+    },
+
+    "LOGIN" : {
+        "FIELD_HEADER_GUAC_DUO_SIGNED_RESPONSE" : "",
```
Same, here, as in the CAS module - you can leave this out.

necouchman commented on extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/resources/translations/ru.json:
```diff
@@ -3,10 +3,110 @@
+        "FIELD_HEADER_GUACD_ENCRYPTION" : "Шифрование:",
+        "FIELD_HEADER_GUACD_PORT"       : "Порт:",
+
+        "FIELD_OPTION_GUACD_ENCRYPTION_EMPTY" : "",
```
This can be left out

necouchman commented on extensions/guacamole-auth-openid/src/main/resources/translations/ru.json:
```diff
@@ -0,0 +1,12 @@
+    },
+
+    "LOGIN" : {
+        "FIELD_HEADER_ID_TOKEN" : "",
```
You can leave this line out.

necouchman commented on extensions/guacamole-auth-radius/src/main/resources/translations/ru.json:
```diff
@@ -0,0 +1,13 @@
+
+    "LOGIN" : {
+        "FIELD_HEADER_GUAC_RADIUS_CHALLENGE_RESPONSE" : "",
+        "FIELD_HEADER_GUAC_RADIUS_STATE"              : "",
```
These two lines can be left out.

necouchman commented on extensions/guacamole-auth-totp/src/main/resources/translations/ru.json:
```diff
@@ -0,0 +1,34 @@
+    },
+
+    "LOGIN" : {
+        "FIELD_HEADER_GUAC_TOTP" : ""
```
This line can be left out.

darksimpson commented:
Hi. Fixed it (I think so).

necouchman commented on guacamole/src/main/webapp/translations/ru.json:
```diff
@@ -185,13 +228,14 @@
         "TABLE_HEADER_HISTORY_USERNAME" : "Имя пользователя",
         "TABLE_HEADER_HISTORY_START"    : "Время начала",
         "TABLE_HEADER_HISTORY_DURATION" : "Продолжительность",
+        "TABLE_HEADER_HISTORY_REMOTEHOST" : "Удаленный узел",
```
Nitpick: It would be good to line these up :-).

necouchman commented on guacamole/src/main/webapp/translations/ru.json:
```diff
@@ -229,11 +274,16 @@
         "ACTION_SAVE"        : "@:APP.ACTION_SAVE",
 
+        "DIALOG_HEADER_CONFIRM_DELETE" : "Удалить профиль расшаривания",
         "DIALOG_HEADER_ERROR" : "@:APP.DIALOG_HEADER_ERROR",
```
Maybe line these up?

darksimpson commented:
Done.

necouchman commented:
Sorry for the delay...a couple more things to fix...
* Your commit messages all need to be tagged with "GUACAMOLE-1054: " at the beginning of the message.
* I ran the translation check tool (`guacamole/util/check-translation.py`) and it turned up just a couple of issues.  There are some missing translations - which is completely fine, you don't have to translate everything, but might just be worth looking over to see if there is anything else you can add - and then a couple of extra fields that shouldn't be in the translation files.  Report is below.  I removed the stuff from the report that wasn't relevant (like blank fields for some of the ticket headers and such).

```
$ find . -type f -iname ru.json -print -exec ./guacamole/util/check-translation.py {} \;
./guacamole/src/main/webapp/translations/ru.json
Original language: en (English)
Translation language: ru (Русский)

The following strings are missing from the translation and should be added:

    APP.TEXT_UNTRANSLATED
    CLIENT.FIELD_PLACEHOLDER_FILTER
    CLIENT.NAME_KEY_ALT
    CLIENT.NAME_KEY_CTRL
    CLIENT.NAME_KEY_ESC
    CLIENT.NAME_KEY_TAB
    PROTOCOL_KUBERNETES.FIELD_OPTION_BACKSPACE_127
    PROTOCOL_KUBERNETES.FIELD_OPTION_BACKSPACE_8
    PROTOCOL_KUBERNETES.FIELD_OPTION_BACKSPACE_EMPTY
    PROTOCOL_KUBERNETES.FIELD_OPTION_COLOR_SCHEME_EMPTY
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_10
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_11
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_12
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_14
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_18
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_24
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_30
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_36
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_48
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_60
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_72
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_8
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_9
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_96
    PROTOCOL_KUBERNETES.FIELD_OPTION_FONT_SIZE_EMPTY
    PROTOCOL_RDP.FIELD_HEADER_DISABLE_COPY
    PROTOCOL_RDP.FIELD_HEADER_DISABLE_PASTE
    PROTOCOL_RDP.FIELD_OPTION_COLOR_DEPTH_EMPTY
    PROTOCOL_RDP.FIELD_OPTION_RESIZE_METHOD_EMPTY
    PROTOCOL_RDP.FIELD_OPTION_SECURITY_EMPTY
    PROTOCOL_RDP.FIELD_OPTION_SECURITY_VMCONNECT
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_DA_DK_QWERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_DE_CH_QWERTZ
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_DE_DE_QWERTZ
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_EMPTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_EN_GB_QWERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_EN_US_QWERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_ES_ES_QWERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_ES_LATAM_QWERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_FAILSAFE
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_FR_BE_AZERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_FR_CH_QWERTZ
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_FR_FR_AZERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_HU_HU_QWERTZ
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_IT_IT_QWERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_JA_JP_QWERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_PT_BR_QWERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_SV_SE_QWERTY
    PROTOCOL_RDP.FIELD_OPTION_SERVER_LAYOUT_TR_TR_QWERTY
    PROTOCOL_RDP.NAME
    PROTOCOL_RDP.SECTION_HEADER_CLIPBOARD
    PROTOCOL_RDP.SECTION_HEADER_REMOTEAPP
    PROTOCOL_RDP.SECTION_HEADER_SFTP
    PROTOCOL_SSH.FIELD_HEADER_DISABLE_COPY
    PROTOCOL_SSH.FIELD_HEADER_DISABLE_PASTE
    PROTOCOL_SSH.FIELD_OPTION_BACKSPACE_127
    PROTOCOL_SSH.FIELD_OPTION_BACKSPACE_8
    PROTOCOL_SSH.FIELD_OPTION_BACKSPACE_EMPTY
    PROTOCOL_SSH.FIELD_OPTION_COLOR_SCHEME_EMPTY
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_10
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_11
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_12
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_14
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_18
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_24
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_30
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_36
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_48
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_60
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_72
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_8
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_9
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_96
    PROTOCOL_SSH.FIELD_OPTION_FONT_SIZE_EMPTY
    PROTOCOL_SSH.FIELD_OPTION_TERMINAL_TYPE_ANSI
    PROTOCOL_SSH.FIELD_OPTION_TERMINAL_TYPE_EMPTY
    PROTOCOL_SSH.FIELD_OPTION_TERMINAL_TYPE_LINUX
    PROTOCOL_SSH.FIELD_OPTION_TERMINAL_TYPE_VT100
    PROTOCOL_SSH.FIELD_OPTION_TERMINAL_TYPE_VT220
    PROTOCOL_SSH.FIELD_OPTION_TERMINAL_TYPE_XTERM
    PROTOCOL_SSH.FIELD_OPTION_TERMINAL_TYPE_XTERM_256COLOR
    PROTOCOL_SSH.NAME
    PROTOCOL_SSH.SECTION_HEADER_CLIPBOARD
    PROTOCOL_SSH.SECTION_HEADER_SFTP
    PROTOCOL_TELNET.FIELD_HEADER_DISABLE_COPY
    PROTOCOL_TELNET.FIELD_HEADER_DISABLE_PASTE
    PROTOCOL_TELNET.FIELD_OPTION_BACKSPACE_127
    PROTOCOL_TELNET.FIELD_OPTION_BACKSPACE_8
    PROTOCOL_TELNET.FIELD_OPTION_BACKSPACE_EMPTY
    PROTOCOL_TELNET.FIELD_OPTION_COLOR_SCHEME_EMPTY
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_10
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_11
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_12
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_14
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_18
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_24
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_30
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_36
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_48
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_60
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_72
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_8
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_9
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_96
    PROTOCOL_TELNET.FIELD_OPTION_FONT_SIZE_EMPTY
    PROTOCOL_TELNET.FIELD_OPTION_TERMINAL_TYPE_ANSI
    PROTOCOL_TELNET.FIELD_OPTION_TERMINAL_TYPE_EMPTY
    PROTOCOL_TELNET.FIELD_OPTION_TERMINAL_TYPE_LINUX
    PROTOCOL_TELNET.FIELD_OPTION_TERMINAL_TYPE_VT100
    PROTOCOL_TELNET.FIELD_OPTION_TERMINAL_TYPE_VT220
    PROTOCOL_TELNET.FIELD_OPTION_TERMINAL_TYPE_XTERM
    PROTOCOL_TELNET.FIELD_OPTION_TERMINAL_TYPE_XTERM_256COLOR
    PROTOCOL_TELNET.NAME
    PROTOCOL_TELNET.SECTION_HEADER_CLIPBOARD
    PROTOCOL_VNC.FIELD_HEADER_DISABLE_COPY
    PROTOCOL_VNC.FIELD_HEADER_DISABLE_PASTE
    PROTOCOL_VNC.FIELD_HEADER_USERNAME
    PROTOCOL_VNC.FIELD_OPTION_CLIPBOARD_ENCODING_CP1252
    PROTOCOL_VNC.FIELD_OPTION_CLIPBOARD_ENCODING_EMPTY
    PROTOCOL_VNC.FIELD_OPTION_CLIPBOARD_ENCODING_ISO8859_1
    PROTOCOL_VNC.FIELD_OPTION_CLIPBOARD_ENCODING_UTF_16
    PROTOCOL_VNC.FIELD_OPTION_CLIPBOARD_ENCODING_UTF_8
    PROTOCOL_VNC.FIELD_OPTION_COLOR_DEPTH_EMPTY
    PROTOCOL_VNC.FIELD_OPTION_CURSOR_EMPTY
    PROTOCOL_VNC.NAME
    PROTOCOL_VNC.SECTION_HEADER_REPEATER
    PROTOCOL_VNC.SECTION_HEADER_SFTP
    SETTINGS_CONNECTION_HISTORY.FILENAME_HISTORY_CSV
    SETTINGS_PREFERENCES.HELP_LOCALE

The following strings are either NOT defined for the original language or are expected to be inherited from the original language and should be removed:

    SETTINGS_PREFERENCES.HELP_LANGUAGE

130 error(s) total.
./extensions/guacamole-auth-radius/src/main/resources/translations/ru.json
Original language: en (None)
Translation language: ru (None)

The following strings are either NOT defined for the original language or are expected to be inherited from the original language and should be removed:

    LOGIN.INFO_RADIUS_ADDL_REQUIRED

1 error(s) total.
```

darksimpson commented:
I've checked that outout log, added some missing translations and also fixed one, thanks. Other strings do not need to be included in translation files because it is either empty or simply doesn't needs to be translated. Also modified commit messages and prepended it with issue ID.
